ColabTool/Shared/PostFooter/index.jsx

```diff
@@ -71,9 +83,42 @@ const renderApproval = ({
   return null;
 };
 
+const renderIcon = (isPastDue) => {
+  if (isPastDue) {
+    return (
+      <WarningIcon color={'torchRed'} />
+    );
+  }
+  return (
+    <ClockIcon />
+  );
+};
+
+const renderText = ({
+  draftDetails,
+  isPastDue,
+}) => {
+  let text = (
```

Comment:
you might be able to save a variable and a few lines of duplicate code with something like this:

```js
const renderText = ({
  draftDetails,
  isPastDue,
}) =>
  <span className={style['post-action']}>
    <Text size={'small'} color={isPastDue ? 'red' : undefined}>{draftDetails.postAction}</Text>
    {isPastDue ? <Text size={'small'}> - Please reschedule or delete.</Text> : null}
  </span>;
```